deploy-manage/security/secure-your-elastic-cloud-enterprise-installation/manage-security-certificates.md

```diff
@@ -31,6 +31,9 @@ Proxy certificate
 
     After the certificates have been installed, connecting securely to {{es}}, {{kib}}, and the Cloud UI or making secure RESTful API calls to ECE should not result in any security warnings or errors.
 
+Adminconsole certificate
+:   Used to connect securely to make RESTful API calls.
```

Comment:
I don't think this is right

The Cloud UI certificate is for API and UI calls

The Adminconsole certificate allow secure connection to an alternative API port that can be used in  incidents scenario where the UI is down (very rare). We recommend re-using the UI certificate for this.